packages/ui/src/Avatar/index.tsx

```diff
@@ -6,12 +6,14 @@ import common from '../styles/common.module.css';
 import {ISize} from '@eruditorgroup/profi-toolkit';
 
 export interface AvatarProps
-  extends Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> {
+  extends Omit<React.HTMLAttributes<HTMLImageElement>, 'children'> {
   design?: 'circle' | 'rect';
   size?: ISize;
   isOnline?: boolean;
   src?: string;
+  alt?: string;
```

Comment:
он уже есть в `React.HTMLAttributes<HTMLImageElement>,`
```suggestion
  alt?: string;
```